docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -60,18 +60,52 @@ If a device is connected over Bluetooth or Wi-Fi and registers an application, S
 !!!
 
 
+###The below describes the behavior related to RSDL functionality only.   
+
+_**Changes**_   
+  1.	New appHMIType: REMOTE_CONTROL was added: all and any remote-control applications must register with this type.   
+  2.	Registration of identical applications (that is, with the same name, same VRSynonyms, same policies_appID), one from driver's device and another one from passenger's device, is allowed by RSDL.   
+  3.	Passenger's applications are not allowed to be activated to FULL level.   
+  
+_**Behavior**_
+
+I. _Application from driver's device registers_ - the vehicle must:   
+     1.	Know the device rank is driver's (see also [RC.OnDeviceRankChanged]).   
+     2.	Mark each application registered from driver's device with "D - \<appName>" when displaying in the list of registered applications (all required information comes from _BasicCommunication.OnAppRegistered_).   
+     3.	Send _SDL.ActivateApp_ to RSDL in case the driver activates such application through either UI or VR.   
+     4.	Send _OnExitApplication(USER_EXIT)_ in case the driver decides to exit such application through either UI or VR.
+
+[RC.OnDevcieRankChanged]: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/feature/docs/RC/RC.OnDeviceRankChanged/index.md
+
+
+II. _Application from passenger's device registers_ - the vehicle must:
```

Comment:
What is the scope of "the vehicle" in this case? Is it referring to the module's responsibility outside of Core?